### ISSUE TITLE
About hyper-parameters

### ISSUE THREAD

**Issue by LaFeuilleMorte** [city-super/Octree-AnyGS#6]
Hi, I noticed this version of octree-gs has different hyperparameters:

![image](https://github.com/user-attachments/assets/5bd835c1-683c-4035-bdfd-109262eeaed2)

And in the default yaml:

![image](https://github.com/user-attachments/assets/bcfb9d26-31ce-4e90-b2e2-c1e28e94156f)

the lambda_normal and lambda_dist are set to 0 by default. Doesn't this help to improve the metrics? 

BTW, if I add depth loss with the mono depth images, Will it help to improve the quality? Looking forward to your reply!

**Comment by jianglh-WHU** [comment]
1. we follow 2DGS to use lambda_normal and lambda_dist, but these losses will influence visual quality, you can certainly ablate them if needed.
2. depth loss will improve the quality.
